RollbarNotifier/Sources/RollbarNotifier/RollbarLogger.m

```diff
@@ -326,7 +326,9 @@ - (void)processSavedItems {
     [fileHandle closeFile];
 
     if (!fileLength) {
-        RollbarSdkLog(@"Processing saved items: no queued items in the file!");
+        if (NO == self.configuration.developerOptions.suppressSdkInfoLogging) {
```

Comment:
```suggestion
        if (self.configuration.developerOptions.suppressSdkInfoLogging == NO) {
```

or

```suggestion
        if (!self.configuration.developerOptions.suppressSdkInfoLogging) {
```

My vote for the latter.

Reply:
@bxsx , both approaches are syntactically identical i would prefer the current one as it is more readable and safer (as it prevents accidental modification of `self.configuration.developerOptions.suppressSdkInfoLogging == NO` into `self.configuration.developerOptions.suppressSdkInfoLogging = NO` - i have my old C-programming habits still embedded in me :)

Reply:
Point taken! ;-)

The lexical grammar of Python is strongly inspired by natural languages, so I'm used to keep expressions as natural as possible as this is definitely the most readable version for any human being. This also ensures to keep a perception on high level (already developed by human languages).

The former suggestion tries to solve both by keeping focus on actual object (left side of expression) with natural grammar in mind.
The latter is a similar approach, however `!` operator is not derived from any of natural languages, so I'm not happy about this too.

As a side note, In Python this problem is solved by using natural operators: `if not self.configuration.developerOptions.suppressSdkInfoLogging:`. This solution resolves all problems here:
- it's written in natural language
- the main subject of the expression is in a right place (left side of the expression)
- it's secure as you can't accidentally modify neither `!` nor `==`


Anyway, as long as this is a pattern across the entire project, I am completely fine with keeping your version here! 💪 So, I'm marking it as resolved.